token-service/src/main/java/org/p2p/token/service/interactor/TokenServiceInteractor.kt

```diff
@@ -35,6 +35,8 @@ internal class TokenServiceInteractor(
         }
     }
 
+    fun getTokensPriceFlow(networkChain: TokenServiceNetwork) = priceLocalRepository.attachToTokensPrice(networkChain)
```

Comment:
```suggestion
    fun getTokenPricesFlow(networkChain: TokenServiceNetwork) = priceLocalRepository.attachToTokensPrice(networkChain)
```
you return flow of prices for any given token